packages/client/styles/code.css

```diff
@@ -6,10 +10,6 @@ html:not(.dark) .shiki-dark {
   display: none;
 }
 
-::-webkit-scrollbar {
-  width: 0px;
-}
```

Comment:
What's this change is about?

Reply:
This code hides the scrollbar.
And I think this is pretty bad, especially when the code overflows.
I you keep this code:
![with](https://user-images.githubusercontent.com/5246045/211496755-2269c9ce-9bf0-43c2-b14e-0a772c01d78c.png)

If you remove it:
![without](https://user-images.githubusercontent.com/5246045/211496785-7f5d4446-f92a-487f-b8e7-1c5b6e21fe6d.png)